_po/ja/reference/modules/pgroonga-standby-maintainer.po

```diff
@@ -0,0 +1,151 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid ""
+"---\n"
+"title: pgroonga_standby_maintainer module\n"
+"upper_level: ../\n"
+"---"
+msgstr ""
+
+msgid "# `pgroonga_standby_maintainer` module"
+msgstr ""
+
+msgid "Since 2.4.2."
+msgstr "2.4.2で追加。"
+
+msgid "This is still an experimental feature."
+msgstr "まだ実験的な機能です。"
+
+msgid "## Summary"
+msgstr "## 概要"
+
+msgid ""
+"The `pgroonga_standby_maintainer` module provides executing [the `pgroonga_wal"
+"_apply()` function][pgroonga-wal-apply] and [the `pgroonga_vacuum()` function]"
+"[pgroonga-vacuum] automatically on stadnby database."
+msgstr ""
+"`pgroonga_standby_maintainer` モジュールは、 [`pgroonga_wal_apply()` 関数][pgroonga-wal"
+"-apply] と [`pgroonga_vacuum()` 関数][pgroonga-vacuum] をスタンバイのデータベース上で自動的に実行します。"
+
+msgid ""
+"Normally, if we use PGroonga with streaming replication, PGroonga's WAL doesn'"
+"t apply on a standby database.\n"
+"Therefore, for example, a first \"SELECT\" after we switch standby database to p"
```

Comment:
```
Therefore, for example, a first \"SELECT\" may become slower after we switch to a primary database from a standby database.  
```
Always S and V should be together then add the explaining sentence. 
もしくは、ここでは、Primary database上でやる最初の \"SELECTE"\のはなしで、WALをPrimaryにapplyする時間が必要という話だと思うので以下でもいいかもです。

```
Therefore, for example, a first \"SELECT\" on a primary database may become slower after we switch from a standby database. Because it takes time to apply PGroonga's WAL to the primary database.
``` 

2度目のPrimary Databaseを"the"にすることで、遅くなるかもしれないprimary databaseを意図しています。